### PR TITLE
Expose current Aedes version, used in aedes-stats

### PR DESCRIPTION
**Pull request by gnought**

### CODE REVIEW

**Comment by mcollina** [comment]
Also, can you add a unit test?

**Comment by gnought** [comment]
I think the unit test should be in aedes-stats project.
I will add the code
`
    doPub('version', aedesInstance.constructor.version)
` 
in aedes-stats `iterate()` function https://github.com/mcollina/aedes-stats/blob/master/stats.js#L52

**Comment by mcollina** [comment]
This should just check that the version exposed is the same one of the
package.json.

Il giorno dom 11 ago 2019 alle 10:05 Gnought <notifications@github.com> ha
scritto:

**Comment by gnought** [comment]
Add to prototype could be, but it will require an extra memory for storing version in each instance.
Version won’t change in each instance, except a user packs two aedes package. In this case he still can get different version using aedes.constructor.version, right?

**Comment by gnought** [comment]
My idea is also be memory-wise

**Comment by mcollina** [comment]
The following will not cause any addition memory allocation:

```js
Aedes.prototype.version =  require('./package').version
```

The reason for that is that it will stick the value on the prototype itself. Instances will look the value there, much like any additional function.

File: test/meta.js [hunk 1]
```diff
@@ -229,6 +229,17 @@ test('emits client', function (t) {
   })
 })
 
+test('get aedes version', function (t) {
+  t.plan(0)
+
+  var broker = aedes()
+  if (broker.version === undefined) {
+    t.fail('version undefined')
```
Comment (mcollina) [inline test/meta.js]: can you check if the content is the same of package.json?

**Pull request merged**